### DISCUSSION TITLE
HSB Issue

### DISCUSSION THREAD

**Discussion by Wikibear** [Koenkk/zigbee2mqtt#9205] [Devices]
Using 1.21.2 and have an issue with HSB brightness will be ignored. I founded an issue at Apr and seems to be fixed?!
![grafik](https://user-images.githubusercontent.com/18314278/137720109-aaf93c1c-3135-4284-a6d3-fd71355a87c0.png)
This bulb is in use:
https://www.zigbee2mqtt.io/devices/404000_404005_404012.html
As you can see in picture brightness will not set half of illumination and stay on maximum.

**Comment by Wikibear** [comment]
No one with same issue?

**Comment by Quautiputzli** [comment]
I have the same problem.
I am still using version 1.18. because its the last one that will work without this problem.

Will there be a solution? In my case only phillips lamps have this problem.
